NEWS.md

```diff
@@ -1,5 +1,14 @@
 # rtweet ...
 
+- `create_token()` has been deprecated in favour of the combination of
+  `rtweet_user()`/`rtweet_bot()`/`rtweet_app()` + `auth_use()` + `auth_save()`.
+
+- `get_token()` and `get_tokens()` have been deprecated in favour of 
+  `auth_get()`.
```

Comment:
`auth_get` is internal. I think this should be `auth_as`

Reply:
`auth_get()` retrieves the current auth; it's not something that most users will need.